jinahub/segmenters/VADSpeechSegmenter/tests/integration/flow.yml

```diff
@@ -0,0 +1,9 @@
+jtype: Flow
+version: '1'
+with:
+  py_modules:
```

Comment:
Can you move this to the executor (under `metas`)?

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/97/commits/d6a5445f807522ab569712f843bc276be3b46a34) for the change.